### PR TITLE
Elastic source for plot

### PR DESCRIPTION
**Pull request by WesleyYep**
to test:  

`sh lambda.sh`
`sh lex.sh deploy`

Go to the test bot:  
"the plot is XXXXXXXX"  
eg. "the plot is a jedi blows up a death star"

### CODE REVIEW

**Comment by WesleyYep** [comment]
files of interest:
- sourceDispatcher
- ElasticSource.js
- moviebotFunction.js

File: lambda/package.json [hunk 1]
```diff
@@ -7,10 +7,11 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
+    "aws4": "^1.6.0",
```
Comment (Wal8800) [inline lambda/package.json]: is this for aws-sdk ? cause i think lambda have aws-sdk by default

Comment (WesleyYep) [reply lambda/package.json]: oh yeah I was testing with it but didn't use it in the end. I'll remove

**Review by Wal8800 — APPROVED** [review]
Looks good to me

**Pull request merged**